jobs/gradle.yml

```diff
@@ -0,0 +1,50 @@
+# gradle
+#
+# Job responsible for building Gradle Wrapper based projects.
+#
+
+parameters:
+  jdkVersions:
+    - name: JDK_8
+      version: "1.8"
+    - name: JDK_11
+      version: "1.11"
+  jobs:
+    - name: Ubuntu_16
+      vmImage: ubuntu-16.04
+    - name: Ubuntu_18
+      vmImage: ubuntu-18.04
+    - name: macOS_10_14
+      vmImage: macos-10.14
+    - name: Windows_2016
+      vmImage: vs2017-win2016
+
+jobs:
+  - ${{ each job in parameters.jobs }}:
+      - job: ${{ job.name }}
+        strategy:
+          matrix:
+            ${{ each jdkVersion in parameters.jdkVersions }}:
+              ${{ jdkVersion.name }}:
+                jdk.version: ${{ jdkVersion.version }}
+        pool:
+          vmImage: ${{ job.vmImage }}
+        steps:
+          - task: Gradle@2
+            inputs:
+              workingDirectory: ""
+              gradleWrapperFile: "gradlew"
```

Comment:
according to the documentation of Gradle@2

> The location in the repository of the gradlew wrapper used for the build. For agents on Windows (including Microsoft-hosted agents), you must use the **gradlew.bat** wrapper. Agents on Linux or macOS can use the gradlew shell script.
So I would expect this config to fail on Windows environment. Unless PowerShell or new Windows Terminal is used, which can handle same way to launch commands as in Linux/MacOS.
source: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/gradle?view=azure-devops

Reply:
strangely enough, the task did work fine: https://dev.azure.com/tomtomweb/GitHub-TomTom-International/_build/results?buildId=11664

Reply:
Strange, apparently their doc is as well wrong as the implementation does this:

https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/GradleV2/gradletask.ts#L120

```
if (isWindows) {
        // append .bat extension name on Windows platform
        if (!script.endsWith('bat')) {
            tl.debug('Append .bat extension name to gradlew script.');
            script += '.bat';
        }
```